packages/orchestration/src/exos/remote-chain-facade.js

```diff
@@ -158,12 +158,18 @@ const prepareRemoteChainFacadeKit = (
          */
         query(msgs) {
           return asVow(() => {
+            /**
+             * @typedef {{
+             *   remoteChainInfo: CosmosChainInfo;
+             *   connectionInfo?: IBCConnectionInfo;
+             * }} QueryState
+             */
             const {
               /** @type {CosmosChainInfo} */
               remoteChainInfo: { icqEnabled, chainId },
               /** @type {IBCConnectionInfo | undefined} */
               connectionInfo,
-            } = /** @type {any} */ (this.state);
+            } = /** @type {QueryState} */ (this.state);
```

Comment:
The `/** @type` annotations within the `const` declaration are likely now redundant.